### ISSUE TITLE
[fvt]2.12.2 xcatprobe need add fullmsg Option in usage

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1672]
env:all os
build:latest build
How to reproduce:
```
[root@c910f02c01p13 ~]# xcatprobe -h
Usage:
xcatprobe -h
xcatprobe -l
xcatprobe [-V] <subcommand>  <attrbute_to_subcommand>

Options:
    -h : get usage information of xcatprobe 
    -l : list all valid sub commands
    -V : print verbose information of xcatprobe
[root@c910f02c01p13 ~]# 
---------------------------------> should add -w Options here.


```

**Comment by hu-weihua** [comment]
Fixed this bug by pull request #1701, @junxiawang , could you  help to verify it? thanks

**Comment by junxiawang** [comment]
verified
